ui/equipment/equipment.scss

```diff
@@ -1,110 +1,77 @@
 /* -------- Equipment --------- */
-
-/* overview section */
-
-.weapons-overview div[data-groupname] span {
```

Comment:
Managing styles across tabs is not really easy.
Instead, we can use macros and variables in `scss` if we need some consistencies.
This section is now moved to the `overviewTab.scss` and restricted to `.weapons-overview` class parent.